changelog/unreleased/2970

```diff
@@ -0,0 +1,9 @@
+Enhancement: optimize fileChanged check
+
+We've moved the fileChanged check up in the code to
```

Comment:
I'd suggest something like this:
```
During a backup restic skips processing files which have not changed since the last backup run.
Previously this required opening each file once which can be slow on network filesystems. The
backup command now checks for file changes before opening a file. This considerably reduces
the time to create a backup on network filesystems.
```